### PR TITLE
#11 updated project name in package.json

### LINKED ISSUE #11: Rename the app in package.json
Rename the app in package.json, after you renamed the project the name in package.json hasn't been updated to match

### PR DESCRIPTION
**Pull request by RomanHyde**
Changed name in package.json to "Pathfinder".
Thank you for spotting that, Neil!

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-  "name": "new-project",
+  "name": "Pathfinder",
```
Comment (nmunro) [inline package.json]: I *think* it has to be all lower case

**Review by nmunro — CHANGES_REQUESTED** [review]
Remember, just commit a change and push on top of this branch and it'll update the pull request :)

File: package.json (hunk 1 above)
Comment (RomanHyde) [reply package.json]: Oh I see! Okay, I'll get that fixed. Thank you!

Comment (RomanHyde) [reply package.json]: It should be fixed now.

**Review by RomanHyde — COMMENTED** [review]
I've changed it now thank you!

**Review by nmunro — APPROVED** [review]
Thanks for reworking it, much appreciated!

**Pull request merged**